### PR TITLE
doc(stack): build and deploy without stack

### PR DESCRIPTION
**Pull request by uguy**
The page to check in preview : 

https://bonitasoft-bonita-continuous-delivery-doc-build_preview-pr-161.surge.sh/bcd/3.4/livingapp_build_and_deploy_no_stack

### CODE REVIEW

**Comment by uguy** [comment]
BCD refactoring in progress, no sure yet if the user will have to extract and use directly the CLI jars.

**Review by florentsarat — APPROVED** [review]

File: modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+Bonita Continuous Delivery lets you easily build and deploy a Bonita Living Applications on a Bonita runtime thanks to the `Bonita Living Application Builder` and the Bonita `Living Application Deployer`, two bonita CLIs wrapped by the `BCD` CLI itself.
+
+We will explain how to easily build and deploy your Bonita Living Application repository without the use of the whole BCD tooling required for stack management. (ex. docker,ansible, terraform, python scripts, yaml scenario files, etc.)
+
+This chapter assumes that:
+
+* you have cloned a Git repository as described in the xref:livingapp_build.adoc[Bonita Living Application Builder] documentation.
+* you already have a Bonita runtime up and running and user credentials for the technical access.
+* you have java (8+) installed on the machine that will perform the build and deploy actions.
```
Comment (rbioteau) [inline modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc]: Java version depends on Bonita version

File: modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+* you have maven (3.3.x) installed on the machine that will perform the build action.
```
Comment (rbioteau) [inline modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc]: 3.6.3 min

File: modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc (hunk 1 above)
Comment (florentsarat) [reply modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc]: java 11 required for the Bonita 2021.2 right?

Comment (uguy) [reply modules/ROOT/pages/livingapp_build_and_deploy_without_docker.adoc]: Correct. "you have java version matching your Bonita server requirements installed" may be better ?

**Pull request merged**